t/admin/global-rules.t

```diff
@@ -411,139 +448,3 @@ GET /t
 passed
 --- no_error_log
 [error]
-
-
-
-=== TEST 12: not unwanted data, PUT
```

Comment:
These tests are still required to test https://github.com/apache/apisix/blob/7c5df5d343ef79f9a292a8eb621fec72280d13ea/apisix/admin/init.lua#L177

Reply:
@spacewander sorry for that and thanks for your guidance. PR renewed with reverting the 12-15 test cases. Ready for review. Any feedback is welcome. Much appreciated.